### DISCUSSION TITLE
Could bento adopt LTS release plan ?

### DISCUSSION THREAD

**Discussion by meistudioli** [ampproject/bento#4] [General]
To whom it may concern:

I am a AMP developer for years. I like web component and enjoy web page could always fetch the latest feature when user visit. Because runtime always fetch newest version and that means some issues occurred might hurt web page's other function.  Is it possible to adopt [Long-Term Stable (lts)](https://amp.dev/documentation/guides-and-tutorials/learn/spec/release-schedule/#long-term-stable-lts) for bento. Let developers decided which version they like ?

**Comment by kvchari** [comment]
Hi thanks for posting this discussion. I think developers already have the ability to decide which versions they want. Bentojs components are currently published to both a CDN (ampcache) and NPM. Developers that want to pin versions of bento components should be able to do so by installing and pinning a specific version of the component from [npm](https://www.npmjs.com/package/@bentoproject/accordion) or an npm mirror (like [unpkg](https://unpkg.com/browse/@bentoproject/accordion@1.2111242025.1/)). If you use a package manager, like `npm`, you can use your package manager to upgrade/change the versions of your components that you're using.
